test/integration/targets/dnf/tasks/main.yml

```diff
@@ -59,10 +59,9 @@
 
 - include_tasks: modularity.yml
   when:
-    - astream_name is defined
     - (ansible_distribution == 'Fedora' and ansible_distribution_major_version is version('29', '>=')) or
       (ansible_distribution in ['RedHat', 'CentOS'] and ansible_distribution_major_version is version('8', '>='))
-    - not dnf5|default('false')
```

Comment:
Same as above but because `'false'` is always "thruthy".

Reply:
Another good catch. The other changes here should be backported through stable-2.13, but this fix will only be needed for stable-2.15. You might want to include it in a separate PR to avoid conflicts when backporting.